pkg/gds/gds.go

```diff
@@ -140,6 +136,19 @@ func (s *GDS) Register(ctx context.Context, in *api.RegisterRequest) (out *api.R
 		log.Warn().Err(err).Msg("ignoring validation error")
 	}
 
+	// Retrieve email address from one of the supplied contacts.
+	var email string
+	if email = getContactEmail(vasp); email == "" {
```

Comment:
Not sure what email we should be using here if multiple contacts have emails, right now it's using the same order defined in other parts of the code.

Reply:
I think that makes sense - we don't have any idea who is submitting the registration, so our best guess is expressed in the order that you specified. Thank you!